### DISCUSSION TITLE
Problems with new Playlists "feature"

### DISCUSSION THREAD

**Discussion by Lady-Fitzgerald** [FreeTubeApp/FreeTube#5049] [General]
The new playlist feature is extremely cumbersome PITA compared to the old one. I don't need, nor want, multiple playlists so I have only one. But to get to that one playlist, I have to click Playlist, then choose the only playlist, then, when it comes up, it's in list view arranged by date starting with the oldest. I've tried to change the sort from newest to oldest but I haven't been able to so I have to scroll all the way down to find the latest. Also, as if list view wasn't enough of a waste of screen space and scroll time, the thumbnails are huge compared to the ones in grid view in subscriptions and half of the screen is wasted with the thumbnail identifying the playlist (as if I can't remember where I am?) when a simple playlist name banner would be sufficient.

Adding injury to injury, clicking on the plus sign in the upper right corner of a thumbnail in subscriptions requires me to waste time selecting the only playlist to put the video into.

I would go back to the previous version but it's been hampered too much by changes in YouTube (bless their paranoid hearts). Is there any way to just bypass all the extra menus where there is only one playlist, get grid view in the play list, and be able to sort the playlist by latest video first?

**Comment by kommunarr** [comment]
We are already in the process of addressing playlist UX concerns for our upcoming releases (#4921, #4929, #4949, #4594, #4605). You should have had your Favorites playlist automatically selected as your Quick Bookmark playlist on startup, which allows you to add videos to your Favorite playlist as easily as before. If it was not set as your Quick Bookmark playlist automatically, you can set it as such [using the steps outlined here](https://github.com/FreeTubeApp/FreeTube/issues/4982#issuecomment-2073361703). Please let us know if it was not set automatically for you, as this should not occur.

As a note, please understand that we're all volunteers working on this project, and multiple feature requests have been made for all of the issues that you have pointed out. I hope you can understand that open source contributors are not very likely to respond in a stately manner when receiving feedback that was delivered in the manner that you have provided it. We're doing the best we can, so we would at the very least request your patience or material support going forward.

**Comment by Lady-Fitzgerald** [comment]
Thanks for your response.
I have no idea if Quick Bookmark was initially enabled at first because I was clicking around trying to sort out the settings that changed with this version and had no idea what Quick Bookmark even was or how to use it. I also did not know that was what the clock was for.
Playlists are still useless for me because I can't change the sort from first added at the top to last added at the top. If I could do that, Playlists would be usable, albeit still clumsy. Adding the ability to display in a grid and getting rid of that unnecessarily gigantic thumbnail identifying the playlist would help immensely.
The reason I came across as harsh to you (BTW, I didn't think I was) is the frustration I experienced with changes that, for me, were serious downgrades and the lack of easily found and understandable documentation. I hate Github because it's so difficult for me to navigate. With my ADHD, I have to have simple, easy to follow, step by step directions which is the opposite of the fragmented structure of GitHub (which I feel is well named). I had a hard time just trying to find where to post my issues. The links you provided appeared to me as having certain changes already done even though they are not.
I know you all are unpaid volunteers but that doesn't lessen the frustration I'm dealing with.
Also, my old playlist was supposed to transfer over but it didn't. I had to do it manually.

__________________________________________________________________________________________________________________


Sent from my computer using a real keyboard!

 

    On Thursday, May 2, 2024 at 01:43:59 PM MST, Jason ***@***.***> wrote:  
 
 


We are already in the process of addressing playlist UX concerns for our upcoming releases (#4921, #4929, #4949, #4594, #4605). You should have had your Favorites playlist automatically selected as your Quick Bookmark playlist on startup, which allows you to add videos to your Favorite playlist as easily as before. If it was not set as your Quick Bookmark playlist automatically, you can set it as such using the steps outlined here. Please let us know if it was not set automatically for you, as this should not occur.

As a note, please understand that we're all volunteers working on this project, and multiple feature requests have been made for all of the issues that you have pointed out. I hope you can understand that open source contributors are not very likely to respond in a stately manner when receiving feedback that was delivered in the manner that you have provided it. We're doing the best we can, so we would at the very least request your patience or material support going forward.
